cmd/state-custom-installer/custom-installer.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+)
+
+const (
+	cmdInstall   = "install"
+	cmdUnInstall = "uninstall"
+)
+
+type Params struct {
+	sourcePath string
+	path       string
+}
+
+func newParams() *Params {
+	return &Params{sourcePath: ".", path: "/tmp"}
+}
+
+func main() {
+	if os.Getenv("VERBOSE") == "true" {
+		logging.CurrentHandler().SetVerbose(true)
+	}
+
+	runErr := run()
+	if runErr != nil {
+		errMsg := errs.Join(runErr, ": ").Error()
+		if locale.IsInputError(runErr) {
+			logging.Debug("state-offline-installer errored out due to input: %s", errMsg)
+		} else {
+			multilog.Critical("state-offline-installer errored out: %s", errMsg)
```

Comment:
Please take care to actually use input errors when an error is caused by user input. Otherwise, we're going to get a lot of superfluous errors reported on our rollbar.

Reply:
Additionally, you'll need to call:

```
rollbar.SetupRollbar("TOKEN")
```

Otherwise error reporting won't work. Seeing as this is an offline installer I'm guessing you might not want error reporting at all, in this case just don't use `multilog` and use `logging` instead.